localization.md

```diff
@@ -3,13 +3,13 @@
 | Language   | User interface | Mapbox Voice API | Android TextToSpeech
 |------------|:--------------:|:----------------:|:---------------------:
 | Catalan    | -              | —                | —
-| Chinese    | -              | ✅               | —
+| Chinese    | -              | ✅               | ✅
 | Danish     | -              | ✅               | ✅
 | Dutch      | -              | ✅               | ✅
 | English    | ✅             | ✅               | ✅
 | French     | -              | ✅               | ✅
 | German     | -              | ✅               | ✅
-| Hebrew     | -              | ✅               | —
+| Hebrew     | -              | -                | —
```

Comment:
This implies that Hebrew voice instructions are unsupported. But in fact the Directions API does produce Hebrew instructions, and [this Stack Overflow answer](https://stackoverflow.com/a/6742772/4585461) implies that at least some Android devices do support Hebrew in the Android TTS engine.

Reply:
@1ec5 I was looking at that answer earlier.  The accepted answer seems incorrect, as it's showing usage of `Locale#getAvailableLocales` which seems unfortunately unrelated to `Locale`s that are actually supported by Android `TextToSpeech`

Reply:
I haven't been able to track down actual confirmation that `TextToSpeech` does support Hebrew